src/proto/parameters.proto

```diff
@@ -50,6 +51,13 @@ message AiControlConfig
     required PlayName override_ai_play = 2 [default = UseAiSelection];
 }
 
+message AiParameterConfig
+{
+    // threshold to decide if ball hasn't been kicked
+    required double ball_is_kicked_m_per_s_threshold = 1
+        [default = 0.3, (bounds).min_double_value = 0.0, (bounds).max_double_value = 5.0];
```

Comment:
`ball.cpp` has a method `Ball::hasBallBeenKicked()`, could you use that instead?

Reply:
May be difficult because sometimes the robot will kick the ball of the wall as a part of the ball placement strategy, so the velocity direction quickly changes. For my FSM, I want the ball to be detected as kicked even after the bounce, so idk if I should use hasBallBeenKicked() since that function checks against a specific velocity direction (if that makes sense)